httputil/retry.go

```diff
@@ -291,3 +292,18 @@ func RetryRequest(endpoint string, doRequest func() (*http.Response, error), rea
 
 	return resp, err
 }
+
+func CertExpiredOrNotValidYet(err error) bool {
```

Comment:
My personal opinion here is that this name is misleading, when I read it i expected it to take a cert or a couple of dates, but instead it's used to check the type of an error. I expected something more in the style like `ShouldRetryError`, but I do see we have other functions named in this style already.

My suggestion would be to append `Error` so it's named `CertExpiredOrNotValidYetError`

Reply:
that's not the golang naming convention though, the hint here is the type of the parameter. os.IsNotExist also doens't say Error

Reply:
But anyway, @mvo5 you can disregard my comment on this suggestion then